HlibSokolovskyi/HW_5/task_2.py

```diff
@@ -0,0 +1,13 @@
+a, b = 0, 1
+
```

Comment:
good job

look at this implementation:
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')